src/Common/src/Interop/Windows/mincore/Interop.Threading.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
+    {
+        [DllImport("api-ms-win-core-synch-l1-1-0.dll", EntryPoint = "CreateEventExW", CharSet = CharSet.Unicode)]
```

Comment:
We have not done pass over CoreRT to apply the same cleanup yet ... https://github.com/dotnet/corert/issues/2241

Since you are touching this file anyway, it would be nice to give it the right shape.